pyrit/datasets/prompt_templates/jailbreak/tom_and_jerry.yaml

```
@@ -0,0 +1,13 @@
+---
+name: TomAndJerry
```

```suggestion
name: tom_and_jerry
```
sorry, nitpicky, but just keeping it somewhat consistent